.rubocop.yml

```diff
@@ -5,7 +5,8 @@ LineLength:
 Style/FrozenStringLiteralComment:
  Enabled: false
 Metrics/BlockLength:
- Exclude:
+  ExcludedMethods: ['describe', 'context']
```

Comment:
Why did you added this? I think I neved had to to that.

Reply:
Also, you dont have any issue with that in this PR 🤔

Reply:
Yeah, I know, that is because I run rubocop on spec and found that XD